go/runtime/bundle/registry.go

```diff
@@ -152,6 +152,37 @@ func (r *Registry) AddManifest(manifest *Manifest, dir string) error {
 	return nil
 }
 
+// Manifests returns all known manifests (regular and detached).
+func (r *Registry) Manifests() []ExplodedManifest {
+	r.mu.RLock()
+	defer r.mu.RUnlock()
+	return slices.Collect(maps.Values(r.manifests))
+}
+
+// RemoveManifest removes a manifest with provided hash.
+func (r *Registry) RemoveManifest(hash hash.Hash) bool {
+	r.mu.Lock()
+	defer r.mu.Unlock()
+	explManifest, ok := r.manifests[hash]
```

Comment:
```suggestion
	manifest, ok := r.manifests[hash]
```
You could shorten this, as from manifest map one expects to get a manifest. And then you can remove `runtimeID` and just use `manifest.ID`.